types/braintree/index.d.ts

```diff
@@ -1685,4 +1685,31 @@ declare namespace braintree {
     export interface TooManyRequestsError extends Error {}
     export interface UnexpectedError extends Error {}
     export interface UpgradeRequired extends Error {}
+
+    /**
+     * Validation errors
+     */
+
+    export interface ValidationError {
+        attribute: string;
+        code: string;
+        message: string;
+    }
+
+    export interface ErrorCollectionsObject {
+        [key: string]: ValidationErrorsCollection;
+    }
+
+    export interface ValidationErrorsObject {
+        [key: string]: ValidationError[];
+    }
+
+    export interface ValidationErrorsCollection {
+        validationErrors: ValidationErrorsObject;
```

Comment:
@MatejSkrbis Where do `validationErrors` and `errorCollections` come into play? Couldn't find them from the docs  and was mostly referencing this page for the PR: https://developers.braintreepayments.com/reference/general/validation-errors/overview/node

Reply:
You've got the point. They should probably be private as they are accessed with the help of `for` and `on` methods. I've removed them.